packages/edit-post/src/store/effects.js

```diff
@@ -72,6 +73,11 @@ const effects = {
 		} );
 	},
 	REQUEST_META_BOX_UPDATES( action, store ) {
+		// Saves the wp_editor fields
+		for ( var i = 0; i < tinyMCE.editors.length; i++ ) {
```

Comment:
so this means you're loading the `tinymce` variable but using the `tinyMCE` global? seems to me like we should replace this with `tinymce` to use the imported variable?

Reply:
We can use the global tinyMCE without importing tinymce at the top.

If we import tinymce and call tinymce.triggerSave or tinymce.editors, they don't work as we would expect to save the extra editors.

Thanks again for the time on this. We're going to fix this issue, which is important for plugins. I'm getting an education in Gutenberg, node, and JS here. I apologize for the confusion.

Reply:
Thanks for your help here :). That's interesting though, now I'm curious. What is the difference between `window.tinymce` and `window.tinyMCE`?

Reply:
window.tinymce and window.tinyMCE seem to be the same object in my console.